assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -235,3 +235,40 @@
 		padding-top: 3px;
 	}
 }
+
+.googlesitekit-settings-notice-ad-blocking-recovery-cta {
+	align-items: center;
+	flex-wrap: wrap;
+	margin-top: 24px;
+	padding: 24px;
+	row-gap: 15px;
+
+	@media (min-width: $bp-tablet) {
+		flex-wrap: nowrap;
+	}
```

Comment:
These styles should be added to the `assets/sass/components/adsense/_googlesitekit-adsense-ad-blocker-recovery.scss` file.

Reply:
Moved to `assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss`